core/src/main/java/com/amplifyframework/api/GraphQlBehavior.java

```diff
@@ -23,14 +23,13 @@
 import com.amplifyframework.api.graphql.GraphQLResponse;
 import com.amplifyframework.api.graphql.MutationType;
 import com.amplifyframework.api.graphql.SubscriptionType;
-import com.amplifyframework.core.ResultListener;
-import com.amplifyframework.core.StreamListener;
+import com.amplifyframework.core.Action;
+import com.amplifyframework.core.Consumer;
 import com.amplifyframework.core.model.Model;
 import com.amplifyframework.core.model.query.predicate.QueryPredicate;
-import com.amplifyframework.hub.HubCategory;
```

Comment:
Are we no longer using the Hub? I thought that we would be publishing everything to hub by default on top of the local callback to caller thread?

Reply:
That was a plan, originally. I think it sounds like a fine idea. The only problem is that we never actually implemented any code to _use_ Hub. We just wrote the Hub and left a bunch of documentation saying that it was getting used, when it wasn't. That was a ~dangerous state, so I've been going through and removing the false documentation. If/when we add Hub integration, we can go back and update the documentation to mention Hub again.